get-secretmanager-secrets/src/client.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GoogleAuth } from 'google-auth-library';
+
+type ClientOptions = {
+  credentials?: string,
+  endpoint?: string,
+}
+
+export class Client {
+  readonly defaultEndpoint: string = 'https://secretmanager.googleapis.com/v1beta1';
+  readonly defaultScope: string = 'https://www.googleapis.com/auth/cloud-platform';
+  readonly userAgent = 'github-action-google-secretmanager/0.1.0';
```

Comment:
Let's keep the userAgent format consistent with the setup-gcloud action: https://github.com/GoogleCloudPlatform/github-actions/blob/66f24ece9d2631f3ce0f06dddf4df0febe9a7eaa/setup-gcloud/src/install-util.ts#L26